tests/monitoring/vm_monitoring.go

```diff
@@ -200,7 +202,7 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 			migration.Annotations = map[string]string{v1.MigrationUnschedulablePodTimeoutSecondsAnnotation: "60"}
 			migration = libmigration.RunMigration(virtClient, migration)
 
-			Eventually(matcher.ThisMigration(migration), 2*time.Minute, 5*time.Second).Should(matcher.BeInPhase(v1.MigrationFailed), "migration creation should fail")
+			Eventually(ThisMigration(migration), 2*time.Minute, 5*time.Second).Should(BeInPhase(v1.MigrationFailed), "migration creation should fail")
```

Comment:
Since you touch it, you can also move it to `.WithTimeout() / .WithPolling()` usage :)

Reply:
Done :)